### DISCUSSION TITLE
1.0.3

### DISCUSSION THREAD

**Discussion by Schwartzblat** [Schwartzblat/WhatsApp-Web-Plus#21] [Announcements]
- Support view once messages of any type.
- Fix bugs.
- Gruntfile added to make contributors life easier. 

<hr /><em>This discussion was created from the release <a href='https://github.com/Schwartzblat/WhatsApp-Web-Plus/releases/tag/1.0.3'>1.0.3</a>.</em>

**Comment by Schwartzblat** [comment]
@talbeerysec thanks you for helping me by finding the bug & a way to fix it ;)

**Comment by Schwartzblat** [comment]
@Blonkyblonk let me know if it fixed your issue.

**Reply by Blonkyblonk** [reply]
GREAATTT 🔥🔥🔥,, I installed version 1.0.4
